api/desecapi/models.py

```diff
@@ -897,20 +897,36 @@ def _act(self):
         self.domain.save(update_fields=['renewal_state', 'renewal_changed'])
 
 
-def captcha_default_content():
-    alphabet = (string.ascii_uppercase + string.digits).translate({ord(c): None for c in 'IO0'})
-    content = ''.join([secrets.choice(alphabet) for _ in range(5)])
+def captcha_default_content(kind: str) -> str:
+    if kind == Captcha.Kind.IMAGE:
+        alphabet = (string.ascii_uppercase + string.digits).translate({ord(c): None for c in 'IO0'})
+        length = 5
+    elif kind == Captcha.Kind.AUDIO:
+        alphabet = string.digits
+        length = 8
+    else:
+        raise ValueError(f'Unknown Captcha kind: {kind}')
+
+    content = ''.join([secrets.choice(alphabet) for _ in range(length)])
     metrics.get('desecapi_captcha_content_created').inc()
     return content
 
 
 class Captcha(ExportModelOperationsMixin('Captcha'), models.Model):
+
+    class Kind(models.TextChoices):
+        IMAGE = 'image'
+        AUDIO = 'audio'
+
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     created = models.DateTimeField(auto_now_add=True)
-    content = models.CharField(
-        max_length=24,
-        default=captcha_default_content,
-    )
+    content = models.CharField(max_length=24, default="")
+    kind = models.CharField(choices=Kind.choices, default=Kind.IMAGE, max_length=24)
```

Comment:
Why 24?

Reply:
it needs some limit. We could do 42 :) Seriously, I chose a multiple of 8 because I found that makes sense. But that's nothing more than an educated guess

Reply:
Aren't the only values "Image" and "Audio"? So I would have thought, the limit is 5.